File: python/3301.py
while True:
    try:
        h, z, l = map(int, input().split())

        if (h > z and z > l) or (h < z and z < l):
            print('zezinho')
        elif (h > l and l > z) or (h < l and l < z):
            print('luisinho')
        else:
            print('huguinho')

    except EOFError:
        break

File: dart/3301.dart
import 'dart:io';

void main() {
  while (true) {
    var input = stdin.readLineSync();

    if (input == null) {
      break;
    }

    List<String> o = input.split(' ');
    int h = int.parse(o[0]);
    int z = int.parse(o[1]);
    int l = int.parse(o[2]);

    if (h > z && z > l || h < z && z < l) {
      print('zezinho');
    } else if (h > l && l > z || h < l && l < z) {
      print('luisinho');
    } else {
      print('huguinho');
    }
  }
}
